web/src/app/modules/shared/services/content/content.service.ts

```diff
@@ -141,6 +155,19 @@ export class ContentService {
     this.current.next(contentResponse);
   }
 
+  private setTitle(response: ContentUpdate) {
+    const title = response.content.title;
+    const titleView = title[title.length - 1] as LinkView;
+    const titleVal = titleView.config.value;
+
+    this.titleService.setTitle(`Octant | ${titleVal} | ${response.namespace}`);
+    this.title.next({
+      namespace: response.namespace,
```

Comment:
The response for non-namespaced scoped items are returning "default" as the namespace. This is a backend issue - either parse the content path to know when not to use the response's namespace or this will be blocked by a new issue to be created.

Reply:
Having assumption for non-namespaced resources should not be a responsibility of fronend and should be addressed by the backend. I'll create an issue for this.